docs/source/column-metadata.rst

```diff
@@ -0,0 +1,185 @@
+***************
+Column Metadata
+***************
+
+This document is heavily inspired by the KDB+ DBMS and its driver language ``q``.
+
+In ``q``, tables are constructed like this:
+
+   .. code-block:: sh
+
+      q) t: ([] name: `Alice`Bob`Joe; amount: 100 200 -400)
```

Comment:
Note to self, always start document with `q` code.  Quickly gets rid of the riff-raff.